sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -615,6 +615,11 @@ def _execute(
             job_config=job_config,
             timeout=self._extra_config.get("job_creation_timeout_seconds"),
         )
+
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
logger.isEOnabled is not necessary, you can just log to the logger
also best practice to domething like


logger.debug("Bigquery job created: %s", job_link)

Reply:
Sounds good. I wanted to be defensive here and not actually generate the job link at all if we're not going to log it. i can redefine the whole log line to inline the `%s` there instead for the same result.

Reply:
If we want to get rid of `logger.isEnabledFor` I'd this instead:
```
logger.debug(f"BigQuery job created: https://console.cloud.google.com/bigquery?project=%s&j=bq:%s:%s", ...)
```
To avoid creating a variable not used outside the logging setting.

Reply:
fixed in https://github.com/TobikoData/sqlmesh/pull/2177/commits/cbdb133c47904d83dcf1cf9206cb031ea28d3b87